target/linux/ramips/dts/DIR-506L.dts

```diff
@@ -0,0 +1,162 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/dts-v1/;
+
+#include "rt5350.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "dlink,dir-506l", "ralink,rt5350-soc";
+	model = "D-Link DIR-506L";
+
+	aliases {
+		led-boot = &led_status_green;
+		led-failsafe = &led_status_red;
+		led-running = &led_status_green;
+		led-upgrade = &led_status_red;
+	};
+
+	gpio-leds {
+		compatible = "gpio-leds";
+
+		usb {
+			label = "dir-506l:green:usb";
+			gpios = <&gpio0 9 GPIO_ACTIVE_LOW>;
+		};
+
+		led_status_green: status_green {
+			label = "dir-506l:green:status";
+			gpios = <&gpio0 19 GPIO_ACTIVE_HIGH>;
+		};
+
+		led_status_red: status_red {
+			label = "dir-506l:red:status";
+			gpios = <&gpio1 1 GPIO_ACTIVE_HIGH>;
+		};
+
+		internet {
+			label = "dir-506l:green:internet";
+			gpios = <&gpio1 0 GPIO_ACTIVE_HIGH>;
+		};
+	};
+
+	gpio-keys-polled {
+		compatible = "gpio-keys-polled";
+		poll-interval = <20>;
+
+		reset {
+			label = "reset";
+			gpios = <&gpio0 2 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+
+		wps {
+			label = "wps";
+			gpios = <&gpio0 7 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;
+		};
+
+		low_battery {
+			label = "low_battery";
+			gpios = <&gpio0 10 GPIO_ACTIVE_HIGH>;
+			linux,code = <BTN_0>;
+		};
+
+		charging {
+			label = "charging";
+			gpios = <&gpio0 11 GPIO_ACTIVE_HIGH>;
+			linux,code = <BTN_1>;
+		};
+
+		on_battery {
+			label = "on_battery";
+			gpios = <&gpio0 12 GPIO_ACTIVE_HIGH>;
+			linux,code = <BTN_2>;
+		};
+	};
+
+	gpio_export {

```

Comment:
@rozhuk-im Please get rid of this gpio_export as well, `regulator-fixed` is probably more appropriate here, for example:

https://github.com/openwrt/openwrt/blob/d0090339117031d1994d139240bb30cb39b73060/target/linux/ramips/dts/MIR3P.dts#L79-L86

For details about gpio-export deprectation please see https://github.com/openwrt/openwrt/pull/1366#issuecomment-424834448